### ISSUE TITLE
URLs or Videos

### ISSUE THREAD

**Issue by thorgod** [apclary/dungeon-revealer#172]
It would be nice to be able to have URLs for the background "image" so that we can use videos like dynamic dungeons that use MP4 

Instead of uploading file, we can past a URL to a video, or image.

**Comment by apclary** [comment]
Good idea. Not sure how easy that would be, I remember @n1ru4l made some changes to the layers for efficiency purposes. If the canvas is just sent as blackness and the image in the background is just on the lower z-layer, it may be very easy.

**Comment by captify-ochekhovskyi** [comment]
https://github.com/dungeon-revealer/dungeon-revealer/pull/1636
